src/Components/Components/test/RendererTest.cs

```diff
@@ -464,12 +496,12 @@ public void CanDispatchTypedEventsToTopLevelComponents()
 
             // Act/Assert: Event can be fired
             var eventArgs = new UIMouseEventArgs();
-            renderer.DispatchEvent(componentId, eventHandlerId, eventArgs);
+            await renderer.DispatchEventAsync(componentId, eventHandlerId, eventArgs);
```

Comment:
As with the comment above, can we avoid the `await`s in these tests so they are making assertions about synchronous behavior?